### ISSUE TITLE
Messy white space

### ISSUE THREAD

**Issue by ghost** [institut-de-genomique/Ultimate-DataTable#20]
I can see that you have some issues with shared code style. Some of you use TAB, others double space. Any chance to get that tided up? Would you like to get pull request with clean formatted code and Eslint rules? I'm using double space and few "easy-to-read" concepts for Angular specific stuff notation.

![image](https://cloud.githubusercontent.com/assets/1313430/11953111/0a057e12-a89e-11e5-8092-923ce0913cb1.png)

BTW. I very much appreciate your work! API is good, SOC is also nice.

**Comment by galbini** [comment]
Yes, thanks for your help. 
The Eslint rules can be integrated with grunt to format automatically the code ?

**Comment by ghost** [comment]
Unfortunately no. But those rules can be used for pre-commit pr pre-merge check. I'm also using them right inside my IDE to spot problems right away.

**Comment by ydeshayes** [comment]
@galbini I can install ESlint and refactor if you want. We can try to migrate the code the ES6, but we'll need to install more tool to downgrade to ES5 when we're shipping.

**Comment by galbini** [comment]
@ydeshayes, good idea for the refactoring but i prefer to wait to have validate your pull request before refactor all the code because the merge will be hard in the other sense.

The upgrade to ES6 can be planned after the next release.
